deploy/examples/travelapp-kogito-travel-agency.yaml

```diff
@@ -17,10 +17,7 @@ spec:
       contextDir: 05-kogito-travel-agency
       reference: override-bootstrap
       uri: https://github.com/ricardozanini/kogito-travel-agency-tutorial
```

Comment:
isn't this example under kiegroup org?

Reply:
Yes, I'll change it.

Reply:
Actually, only on Maciej's repo. Since it's the source, I'll add it.